aperture_properties.py

```diff
@@ -858,7 +864,9 @@ class ApertureProperties(HaloProperty):
             "BHmaxAR",
             "BHmaxlasteventa",
             "com",
+            "com_star",
             "vcom",
+            "vcom_star",
```

Comment:
Isn't the `vcom_star()` function missing?

Reply:
It already exists since it's used in the calculation of some of the other properties